src/components/EpisodeCard/EpisodeCard.js

```diff
@@ -0,0 +1,97 @@
+import LazyLoad from 'react-lazyload'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+EpisodeCard.propTypes = {
+  episode: PropTypes.object.isRequired,
+}
+
+export default function EpisodeCard({ episode }) {
+  const { datePublished, duration } = episode
+  const months = [
+    'Jan.',
+    'Feb.',
+    'Mar.',
+    'Apr.',
+    'May.',
+    'Jun.',
+    'Jul.',
+    'Aug.',
+    'Sep.',
+    'Oct.',
+    'Nov.',
+    'Dec.',
+  ]
+  const date = new Date(datePublished * 1000)
+  const time = Math.floor(duration / 60)
+
+  return (
+    <Card>
+      <ImageAndTitleContainer>
+        <LazyLoad height={48} width={48} offset={96} overflow={true} once>
+          <ImageContainer>
+            <Image
+              src={episode.feedImage}
+              alt={'Poster'}
+              width="48"
+              height="48"
+            />
+          </ImageContainer>
+        </LazyLoad>
+
+        <Title>{episode.title}</Title>
+      </ImageAndTitleContainer>
+      <Description>{episode.description}</Description>
+      <Audio controls={true} src={episode.enclosureUrl}></Audio>
+      <Span>{`${date.getDate()} ${
+        months[date.getMonth()]
+      } ${date.getFullYear()} ● ${time === 0 ? 1 : time} min.`}</Span>
+    </Card>
+  )
+}
+
+const Card = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 0.7rem;
+  padding-bottom: 0.7rem;
```

Comment:
I think it is better practice not using relative values for gap/padding/margin etc. on wider screens it might look not so good.

Reply:
Hm i thought it is always better to use rem over px. Because it scale with users preference. I will look into that again.